setup.py

```diff
@@ -18,10 +18,11 @@ def read(fname):
     "napari-plugin-engine>=0.1.4",
     "numpy",
     "aicssegmentation>=0.1.22",
+    "magicgui >= 0.2.9",
 ]
 
 test_requirements = [
-    "black>=19.10b0",
+    "black==19.10b0",
```

Comment:
What's the reason for this?

Reply:
At least on my computer (and OS and python version), if I install newer versions of black (newest is 20.0b1 or something like that), it doesn't work with mypy.